src/test/java/com/portatlas/ServerTest.java

```diff
@@ -93,24 +96,77 @@ public void testAddPuttRequestAndResponse() {
 
         assertEquals(putResponse.getStatus(), server.router.route(putRequest).getStatus());
     }
+    
+    @Test
+    public void testAddGetFile1AndResponse() {
+        Response getFile1Response = Response.builder()
+                                            .statusCode(StatusCodes.OK)
+                                            .build();
+        Request getFile1Request = new Request(RequestMethod.GET, "/file1" , HttpVersion.CURRENT_VER);
+
+        assertEquals(getFile1Response.getStatus(), server.router.route(getFile1Request).getStatus());
```

Comment:
why did you have to create a getFile1Response? Why not just put StatusCodes.OK as the first argument?